### PR TITLE
adds lifeboat switchover instructions runbook

### PR DESCRIPTION
**Pull request by kilbergr**
# [Tabletop exercise: Lifeboat Switchover #175](https://github.com/OHS-Hosting-Infrastructure/infrastructure/issues/175)

Changes proposed in this pull request:

- Add a runbook detailing how to perform a switchover to use Lifeboat for ECLKC rather than the live ECLKC reader site.

### CODE REVIEW

File: docs/runbooks/lifeboat-switchover.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Lifeboat Switchover
+
+## Context
+
+Lifeboat is the read-only version of the ECLKC Database that we can switch to during maintenance windows and in the case of an incident when the website is getting attacked.
+
+Lifeboat sync jobs run every night around 11pm, so the database should be within 24 hours of up to date at all times. Before switching over, you should check that it successfully ran the night before. It generally takes about 15-20 minutes to sync the databases.
```
Comment (rahearn) [inline docs/runbooks/lifeboat-switchover.md:7]: Does the same sync job copy over to backups in both east and west?

**Comment by rahearn** [comment]
[ question ] @ssatagopan Do these steps seem like something you'd be comfortable running?

**Comment by ssatagopan** [comment]
> [ question ] @ssatagopan Do these steps seem like something you'd be comfortable running?

@rahearn Went through it without executing just now and seems like I could run, if I had the correct permissions. I just have read-only on AWS right now. But I can at least view all the screens mentioned and go through the edit actions without saving.

File: docs/runbooks/lifeboat-switchover.md (hunk 1 above)
Comment (kilbergr) [reply docs/runbooks/lifeboat-switchover.md:7]: There are 2 jobs:
[Project Drupal Production to Lifeboat Sync (All)](https://jenkins.eclkc.info/job/Drupal%20Production%20to%20Lifeboat%20Sync%20(All)/) and 
[Project Drupal Production to Lifeboat-West Sync v2 (All)](
https://jenkins.eclkc.info/job/Drupal%20Production%20to%20Lifeboat-West%20Sync%20v2%20(All)/)

Each one runs every night.

**Review by nateprice18f — APPROVED** [review]
The addition of the Lifeboat failover checklist looks good.

**Pull request merged**